llvm/lib/IR/PassInstrumentation.cpp

```diff
@@ -33,7 +34,10 @@ PassInstrumentationCallbacks::getPassNameForClassName(StringRef ClassName) {
       Fn();
     ClassToPassNameCallbacks.clear();
   }
-  return ClassToPassName[ClassName];
+  auto PassNameIter = ClassToPassName.find(ClassName);
+  if (PassNameIter != ClassToPassName.end())
+    return PassNameIter->second;
+  return {};
```

Comment:
```suggestion
  return "";
```
is a bit clearer, I think.

Reply:
Isn't `StringRef()` different from `StringRef("")`? They'd compare the same, but go through different constructors (no need to check and store the string literal). Not that it'd matter, but at least in other places `{}` is used so I'd go for consistency.